lib/Doctrine/Common/Cache/RedisCache.php

```diff
@@ -39,12 +45,28 @@ public function getRedis()
         return $this->redis;
     }
 
+    /**
+     * Gets the used serializer
+     *
+     * @return int
+     */
+    public function getSerializer()
```

Comment:
Please declare the return type (instead of using doc block)

Reply:
Also, any reason for it be public?

Reply:
Do you mean `public function getSerializer(): ?int` (which only seems to be supported since PHP 7.1)?

Reply:
Yeah, it should unlikely be `public` on the cache instance

Reply:
Hmm, it's `public` to make `RedisCacheTest::testSerializerOptionWithOutIgbinaryExtension` work.

Reply:
Can you change the test bit to use reflection, and then we look at how to make it work without reading this at all?

Reply:
If that's ONLY for testing purposes we should simply not have the method at all... `self::assertAttributeSame()` helps us on that.